PlanningPoker.Tests/ServicesTests.cs

```diff
@@ -15,98 +20,145 @@ namespace PlanningPoker.Tests
     public class ServicesTests
     {
         private readonly IUserService _sut;
-        private readonly Room _room = new Room() { id = "123", name = "Room1", CreatorId = "123456" };
-        private readonly UserConnection _userConnection = new UserConnection() { ConnectionId = "123456", Name = "User" };
-        private readonly UserVote _userVote = new UserVote() { UserName = "User", Vote = 1 };
+        private readonly IUserRepository _userRepository;
+        private readonly IRoomsRepository _roomsRepository;
+        private readonly IHubContext<LoopyHub> _hubContext;
+        private readonly IUnitOfWork _unitOfWork;
+        private const string One = "1";
```

Comment:
Just a minor note, but I Would put constant members first, then all the readonly members